spec.bs

```diff
@@ -600,26 +621,28 @@ partial interface Navigator {
 dictionary AuctionAdConfig {
   required USVString seller;
   required USVString decisionLogicURL;
+
   USVString trustedScoringSignalsURL;
   sequence<USVString> interestGroupBuyers;
   Promise<any> auctionSignals;
-  record<DOMString, DOMString> requestedSize;
-  sequence<record<DOMString, DOMString>> allSlotsRequestedSizes;
   Promise<any> sellerSignals;
   Promise<DOMString> directFromSellerSignalsHeaderAdSlot;
   unsigned long long sellerTimeout;
   unsigned short sellerExperimentGroupId;
-  USVString sellerCurrency;
   Promise<record<USVString, any>> perBuyerSignals;
   Promise<record<USVString, unsigned long long>> perBuyerTimeouts;
   Promise<record<USVString, unsigned long long>> perBuyerCumulativeTimeouts;
+  USVString sellerCurrency;

```

Comment:
DOMString?

Reply:
OK please report back if you can.

Reply:
You were in the discussion in the chat room, and I don't know if there was a conclusion, if I didn't miss anything.

Reply:
I don't recall either. Maybe you could look back and try and summarize it here so we can resolve this one way or another.

Reply:
I don't see a conclusion from that chat, and just confirmed with people it was the case. We found some issues related to DOMString and base::Value, which we need to find a solution but not high priority at this moment. Matt and I agreed that it might be good to keep what we have for now, until we have a final solution for problems related to this.

Reply:
OK, then let's keep the spec PR as-is for now then, since it doesn't seem resolvable right now, and isn't a big deal. Having a tracking issue for this would be great though, so we don't have to re-remember all of this stuff about base::Value and the various USVString usages in the spec.